xml/System.Buffers/ArrayPool`1.xml

```diff
@@ -34,8 +42,8 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Invokes the static constructor when a new instance of the <see cref="ArrayPool{T}"/> class is instantiated.</summary>
```

Comment:
Will do. Good timing, I was just working on adding a description to the ns-System.Buffers.xml file to add to this PR so that you can review it.